### PR TITLE
Update publications.yaml

### PR DESCRIPTION
**Pull request by alirahkay**

### CODE REVIEW

File: _data/publications.yaml [hunk 1]
```diff
@@ -3,6 +3,21 @@
 # No preprint field, means that the paper is published.
 # Publications will be sorted by the date field.
 
+- title: "Towards Evaluating Adaptivity of Model-Based Reinforcement Learning Methods"
+  authors:
+      - Yi Wan\*
```
Comment (AndreasMadsen) [inline _data/publications.yaml:8]: How should this be rendered? The `\` is included in the output, I assume you don't want that. You can just remove `\` if you want `*` to be shown as is.

Skærmbillede 2022-04-26 kl  17 59 48

**Review by AndreasMadsen — CHANGES_REQUESTED** [review]

File: _data/publications.yaml (hunk 1 above)
Comment (alirahkay) [reply _data/publications.yaml:8]: Ah, I thought markdown would handle it. I just wanted to make sure it would be printed as a star.
I'll remove them.

Comment (alirahkay) [reply _data/publications.yaml]: Just realized it's not markdown.

**Review by AndreasMadsen — APPROVED** [review]
LGTM

**Pull request merged**